wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,330 @@
+
+## 2007
+
+*Note: During the period where Ranked Score was used, Rank #1 would change frequently, thus the earlier listings may use months where exact dates aren't available.*
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_AU] [peppy](https://osu.ppy.sh/users/2) | 2007-10-4 | October 2007 |  |
+| ![][flag_AU] [eyup](https://osu.ppy.sh/users/88) | October 2007 | 2007-10-11 |  |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-10-11 | 2007-11-17 |  |
+| ![][flag_NZ] [Echo](https://osu.ppy.sh/users/431) | 2007-11-17 | 2007-11-23 |  |
+| ![][flag_US] [Ivalset](https://osu.ppy.sh/users/827) | 2007-11-23 | 2007-12-24 |  |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2007-12-24 | *2008-09-07* |  |
+
+## 2008
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | *2007-12-24* | 2008-09-07 |  |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | 2008-09-07 | 2008-09-07 |  |
+| ![][flag_US] [Saturos](https://osu.ppy.sh/users/3781) | 2008-09-07 | 2008-10-07 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2008-10-07 | 2008-12-11 |  |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | 2008-12-11 | *2009-01-19* |  |
+
+## 2009
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | *2008-12-11* | 2009-01-19 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-01-19 | 2009-03-05  |  |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | 2009-03-05  | 2009-03-06  |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-03-06 | March 2009  |  |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | March 2009 | 2009-03-18 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-03-18 | 2009-04-16 |  |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | 2009-04-16 | 2009-04-18 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-04-18 | June 2009 |  |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | June 2009 | July 2009 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | July 2009 | 2009-07-05 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2009-07-05 | July 2009 |  |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | July 2009 | 2009-08-13 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2009-08-13 | 2009-09-03 |  |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2009-09-03 | October 2009 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | October 2009 | November 2009 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | November 2009 | *2010-02-07* |  |
+
+## 2010
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *November 2009* | 2010-02-07 |  |
```

Comment:
```suggestion
| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | October 2009 | November 2009 |  |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2010-12-08 | 2010-12-25 |  |
| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 2010-12-25 | *2010-02-07* |  |

## 2010

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | *2010-12-25* | 2010-02-07 |  |
```

I had a prediction Ana_Coppola would have a reign here, and I was percisely correct!

Doomsday achieves rank 1 by 8 December 2009: https://imgur.com/8irCSvO

Ana_Coppola takes rank 1 by 25 December 2009: https://osu.ppy.sh/community/forums/topics/21622?start=276369

H1ko takes rank 1 by 7 February 2010: https://web.archive.org/web/20100207194753/http://osu.ppy.sh/p/playerranking

I think it's highly likely the January 2011 Ana_Coppola reign had the wrong year, and that this reign checks off verifying Ana_Coppola.

Reply:
With sources should be like:
```suggestion
| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | October 2009 | November 2009 |  |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | November 2009 | 2009-12-08 |  |
| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 2009-12-08 | *2010-02-07* | [\[1\]](https://osu.ppy.sh/community/forums/topics/20010?start=262589) |

## 2010

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | *2009-12-25* | 2010-02-07 | [\[1\]](https://osu.ppy.sh/community/forums/topics/20010?start=262589) |
```

H1ko source should go on next line, will add new comment